pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -61,6 +61,8 @@ service RuntimeService {
     // ContainerStatus returns status of the container. If the container is not
     // present, returns an error.
     rpc ContainerStatus(ContainerStatusRequest) returns (ContainerStatusResponse) {}
+    // UpdateContainerResources updates ContainerConfig of the container.
+    rpc UpdateContainerResources(UpdateContainerResourcesRequest) returns (UpdateContainerResourcesResponse) {}
```

Comment:
rather than enumerate a full list of resources, prefer we comment that a runtime should return an error if a resource is changed that cannot be mutated.